packages/devtools_app/lib/src/shared/analytics/metrics.dart

```diff
@@ -39,3 +39,18 @@ class ProfilerScreenMetrics extends ScreenAnalyticsMetrics {
   final int cpuSampleCount;
   final int cpuStackDepth;
 }
+
+class InspectorTreeControllerMetrics extends ScreenAnalyticsMetrics {
```

Comment:
nit: rename to InspectorScreenMetrics so that this can be a general class for all inspector custom metrics.

make parameters optional, and add dart doc for each field.